### ISSUE TITLE
WCAG 4.1.2: Ensures aria-hidden elements are not focusable nor contain focusable elements (.cdk-focus-trap-anchor.cdk-visually-hidden[aria-hidden="true"]:nth-child(2))

### ISSUE THREAD

**Issue by KTottenDC** [DouglasCountyCO/Accessibility-PowerBI-Reports-Public#1]
#### Issue
`Ensures aria-hidden elements are not focusable nor contain focusable elements` ([aria-hidden-focus](https://accessibilityinsights.io/info-examples/web/aria-hidden-focus))
#### Target application
[Foreclosure Search - Power BI](https://app.powerbigov.us/groups/4481de21-0e76-4e03-b12f-c27b430dd50b/reports/3f00ede7-158a-44c2-8d0a-8c300542c84b/ReportSectionffcc13330b770bc22d71)
#### Element path
.cdk-focus-trap-anchor.cdk-visually-hidden[aria-hidden="true"]:nth-child(2)
#### Snippet
`<div tabindex="0" class="cdk-visually-hidden cdk-focus-trap-anchor" aria-hidden="true"></div>`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex="-1" or be removed from the DOM
#### Environment
Microsoft Edge version 125.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.42.0 (axe-core 4.8.4), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
